### ISSUE TITLE
bndtools < 5.3 may create invalid ZIP file extra fields (jar loading exceptions with recent JRE versions)

### ISSUE THREAD

**Issue by aumann** [stempler/bnd-platform#78]
I've just noticed (after creating new Jenkins nodes with up-to-date JREs used for builds) that we get the following exception for one of our bundles created by the bnd-platform plugin (from maven/tycho):
```
Caused by: org.eclipse.tycho.core.osgitools.OsgiManifestParserException: \
   Exception parsing OSGi MANIFEST [...]c.i.e.h.all-3.3.0.v20230901-1110.jar: \
   Invalid CEN header (invalid extra data field size for tag: 0xbfef at 125)
    at org.eclipse.tycho.core.osgitools.DefaultBundleReader.doLoadManifest (DefaultBundleReader.java:69)
```

I believe this is related to: https://bugs.openjdk.org/browse/JDK-8313765 - in this backported fix the JDK team loosened a very recently introduced ZIP stricter header validation. However, they state that even with this relaxed validation, there will still be errors for bundles created by BND < 5.3

The bnd release notes mention the fix here, I believe (`Fixed handling of ZIP file extra field support.`): https://github.com/bndtools/bnd/wiki/Changes-in-5.3.0

So I guess my main question is: is there anything stopping us from updating bnd-lib from 5.2 to 5.3? If it were a matter of "just bumping the version", I could open a pull request.

**Comment by stempler** [comment]
I was working on some fixes anyway and used the opportunity to update bndlib to 6.4.1.
Would be great if you can give the current SNAPSHOT version (3.0.0-SNAPSHOT) a try to see if the problem is solved.

**Comment by aumann** [comment]
Thanks a lot. I'll report back - will need some time, though, as the update from gradle 6 to 7/8 is causing some problems here.

**Comment by jochenberger** [comment]
Looks as if this does indeed fix the issue, at least for me.

**Comment by aumann** [comment]
I've just verified it, too: The error under `JDK 17.0.8.1+1` is present when building with bnd-platform `2.0.1` and gone when building with the current `3.0.0-SNAPSHOT`

**Comment by stempler** [comment]
Thanks a lot for the feedback!
Just released 3.0.0 which is available from Maven Central or the Gradle Plugin Portal.
